foo_060_food_system_emissions/foo_060_food_system_emissions_processing.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import pandas as pd
+import os
+import sys
+utils_path = os.path.join(os.path.abspath(os.getenv('PROCESSING_DIR')),'utils')
+if utils_path not in sys.path:
+    sys.path.append(utils_path)
+import util_files
+import util_cloud
+import util_carto
+from datetime import datetime
+import urllib
+from zipfile import ZipFile
+
+# Set up logging
+# Get the top-level logger object
+logger = logging.getLogger()
+for handler in logger.handlers: logger.removeHandler(handler)
+logger.setLevel(logging.INFO)
+# make it print to the console.
+console = logging.StreamHandler()
+logger.addHandler(console)
+logging.basicConfig(format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+# name of table on Carto where you want to upload data
```

Comment:
is there a reason why the version (i.e. rw0, rw1, etc.) is missing?

Reply:
I will add it :O